chargeOclock/Source/Other/NetworkAPI.swift

```diff
@@ -55,6 +58,29 @@ class NetworkAPI {
 			}.resume()
```

Comment:
indentation issue (not your code, I know, if you refactor the methods, keep an eye out for this one)